src/test/pybind/test_ceph_argparse.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+# use the script to setup the nosetest environment as it should be.
+# Mainly due /usr/bin/env uses a parameter setup different from what
+# python expects
+
+[ ! -z $CEPH_ROOT ] || exit 1
+
+PYBIND=$CEPH_ROOT/src/pybind /usr/local/bin/nosetests  --stop $CEPH_ROOT/src/test/pybind/test_ceph_argparse.py
```

Comment:
@wjwithagen i think the root cause might be that the shebang line in `src/test/pybind/test_ceph_daemon.py` assume that `nosetests` is located in `/usr/bin`. while on your FreeBSD installation, it's not the case.

as to `PYBIND`, it's not necessary. as the cmake function `add_ceph_test()` takes care of the pybinding path already by setting `PYTHONPATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/cython_modules/lib.${PYTHON${PYTHON_VERSION}_VERSION_MAJOR}:${CMAKE_SOURCE_DIR}/src/pybind`.

instead of using `#!/usr/bin/nosetest --nocapture`, probably we can just use `#!/usr/bin/env nosetests`. because shebang line does not support more than two or more parameters. i guess it's safe to remove the `--nocapture` option. it does not hurt to read the captured message sent to stdout along with the error message. @dachary what do you think?